npm-shrinkwrap.json

```diff
@@ -1677,7 +1677,7 @@
         "debug": "^2.6.3",
         "detect-libc": "^1.0.3",
         "fs-extra": "^3.0.1",
-        "node-abi": "^2.0.0",
+        "node-abi": "^2.4.5",
```

Comment:
Super nitpicky, but can you revert this line. I pulled down the patch and this line was reverted.